markets/bfp-market/contracts/storage/Position.sol

```diff
@@ -275,25 +275,45 @@ library Position {
         runtime.positionDecreasing =
             MathUtil.sameSide(currentPosition.size, newPosition.size) &&
             MathUtil.abs(newPosition.size) < MathUtil.abs(currentPosition.size);
-        if (!runtime.positionDecreasing) {
-            // We need discounted margin collateral as we're verifying for liquidation here.
-            //
-            // NOTE: `marginUsd` looks at the current overall PnL but it does not consider the 'post' settled
-            // incurred fees hence get `getNextMarginUsd` -fees.
-            runtime.discountedNextMarginUsd = getNextMarginUsd(
-                marginValuesForLiqValidation.discountedMarginUsd,
-                runtime.orderFee,
-                runtime.keeperFee
-            );
 
-            (runtime.im, runtime.mm) = getLiquidationMarginUsd(
-                newPosition.size,
-                params.oraclePrice,
-                marginValuesForLiqValidation.collateralUsd,
-                marketConfig,
-                addresses
-            );
+        (runtime.im, runtime.mm) = getLiquidationMarginUsd(
```

Comment:
Unrelated to this change but naming variables im and mm is a bit hard to read.
At least Runtime structs should have a comment on the full form of the abbreviations.

Reply:
I'd prefer naming them `initialMargin` and `maintenanceMargin`